### PR TITLE
Show equivalences on the report with scientific annotation

### PR DESCRIPTION
**Pull request by mirpedrol**
image

### CODE REVIEW

**Comment by skrakau** [comment]
https://github.com/nextflow-io/nf-co2footprint/issues/79

**Comment by tillenglert** [comment]
I added a threshold, so that the scientific notation is only used on values greater than 999 and less than 0.001. This improves readability, without showing a 0 at any time.
Also the tests needed to be adjusted to the not rounded values.

**Comment by tillenglert** [comment]
@skrakau and @mirpedrol , You can both take a look again, I think we can ignore the DCO check this time (there is a trigger in the actions), as we both committed to this PR!

File: plugins/nf-co2footprint/src/main/nextflow/co2footprint/CO2FootprintFactory.groovy [hunk 1]
```diff
@@ -748,10 +748,10 @@ class CO2FootprintFactory implements TraceObserverFactory {
             List equivalences = computeCO2footprintEquivalences()
             [ co2:HelperFunctions.convertToReadableUnits(total_co2,3), 
               energy:HelperFunctions.convertToReadableUnits(total_energy,3),
-              car: equivalences[0],
-              tree: equivalences[1],
-              plane_percent: equivalences[2],
-              plane_flights: equivalences[3]
+              car: equivalences[0]?HelperFunctions.convertToScientificNotation(equivalences[0]):equivalences[0],
```
[the remark this answers is not in the document]
Comment (tillenglert) [reply plugins/nf-co2footprint/src/main/nextflow/co2footprint/CO2FootprintFactory.groovy:779]: Thanks for the suggestion! I think using a non custom formatting, will help maintaining the code, so I implemented as suggested (but including the newly set thresholds)

**Comment by tillenglert** [comment]
It is now looking like this: Highest readability for 999≥x≥0.001, all greater or smaller values are converted to scientific notation

image

**Review by skrakau — APPROVED** [review]

**Pull request merged**